azure-pipelines-integration.yml

```diff
@@ -1,77 +1,78 @@
-# Branches that trigger a build on commit
```

Comment:
Commenting out integration tests while I play with thsi cause I don't want to stress our integration machine test pool.